test/local/account_routes.js

```diff
@@ -572,6 +581,8 @@ test('/account/create', function (t) {
       keys: 'true'
     }
   })
+  var clientAddress = '127.0.0.1'
```

Comment:
Any objection to using the value that is mocked in request? https://github.com/mozilla/fxa-auth-server/blob/master/test/mocks.js#L212